NineChronicles.Headless/GraphTypes/StandaloneQuery.cs

```diff
@@ -203,43 +203,6 @@ TransferNCGHistory ToTransferNCGHistory(TxSuccess txSuccess, string memo)
                 }
             );
 
-            Field<NonNullGraphType<LongGraphType>>(
```

Comment:
I guess it could be used for other systems...  how about giving it a grace period(using depreacted) and deleting it after?

Reply:
I agree with you. I will turn it into deprecated status, not deleted.

Reply:
I fixed it in 5e450f6